core/src/main/java/io/neow3j/contract/Hash160.java

```diff
@@ -153,16 +155,15 @@ public static Hash160 fromScript(String script) {
 
     private void checkAndThrowHashLength(byte[] scriptHash) {
```

Comment:
This method still uses the `scriptHash` name in variable and exception message but should use just `hash`.